python/lsst/ts/watcher/alarm.py

```diff
@@ -223,6 +230,42 @@ def close(self):
         self._cancel_escalation_timer()
         self._cancel_unmute()
 
+    async def make_log_entry(self, log_server_url):
+        """Post message to narrative log entry in response to alarm"""
+        now = datetime.datetime.now().isoformat()  # use ts_utils repo?
+        # Required? fields in payload:
+        #   message_text, level, user_id, user_agent, is_human
+        msg = f"alarm:{self.name} severity={self.severity}"
+        user = "make_log_entry"
+        payload = {
+            "message_text": msg,
+            "level": 0,
+            "user_id": "admin@love1.tu.lsst.org",  # TODO
+            "user_agent": user,
+            "is_human": False,
+            "tags": ["watcher", "alarm", "make_log_entry"],
+            "date_begin": now,
+            "date_end": now,
+            # ## Let following fields be defaulted.
+            # 'systems': ['ObservatorySoftwareTools'],
+            # 'subsystems': ['LOVE'],  # TODO
+            # 'cscs': [],
+            # 'urls': [],
+            # 'time_lost': 0,
+        }
+
+        url = f"{log_server_url}/messages"
+        print(f"Posting to log: {url=} {payload=}")  # TODO REMOVE!!!
+
+        # AIOHTTP docs say don't create session per request. We do so anyhow.
+        # Use default timeout (=5 minutes for whole operation)
+        async with aiohttp.ClientSession(raise_for_status=True) as session:
+            async with session.post(url=url, json=payload) as response:
+                post_values: dict = await response.json()
+                dbg_msg = f"Response (json) from Post: {post_values=}"
+                self.log.debug(dbg_msg)
```

Comment:
Any reason for not doing

```
self.log.debug(f"Response (json) from Post: {post_values=}.")
```

?

Reply:
Changed.  But only reason was I was using dbg_msg in both log and print.  I don't know where the log is, so print was easier for me during dev.